apps/public-docsite-v9/src/Concepts/SSR/SSR.stories.mdx

```diff
@@ -195,3 +195,13 @@ const DefaultOpenMenu = () => {
   );
 };
 ```
+
+### Strict mode
+
+Strict mode is currently not supported. To avoid hydration issues, disable strict mode in your app by adding the following configuration to your `next.config.js` file:
```

Comment:
we also have open bugs in strict mode like https://github.com/microsoft/fluentui/issues/24085 or https://github.com/microsoft/fluentui/issues/25232

So the disclaimer should rather be applied to the entire library and not just SSR

Reply:
@ling1726, I think the main difference is that those 2 bugs are specific to Strict Mode in React 18 vs the SSR issues existing in Strict Mode for any version of React.

Reply:
> are specific to Strict Mode in React 18

@khmakoto actually those issues I linked also happen for strict mode in 17 🙈

Reply:
@ling1726 Updated it to be in the quickstart and refer to all repo.